radis/lbl/calc.py

```diff
@@ -523,6 +517,23 @@ def _check_molecules_are_consistent(
             float("inf"),
         ]  # Using a list to store minimum wstep value at 1st index
 
+    flag = False
+    # Check if mole_fraction is dictionary of multiple molecules and have non air diluent, if yes then set value of flag as true and update list of diluents
+    if isinstance(mole_fraction, dict) and isinstance(diluent, dict):
+        flag = True
+        for molecule, fraction in mole_fraction.items():
+            if molecule not in {"CO2", "H2O", "He", "H2"}:
+                import warnings
+
+                warnings.warn(
```

Comment:
Radis uses its own subclass of Warnings, which allows to activate/deactivate them. Here, use a [AccuracyWarning](https://radis.readthedocs.io/en/latest/source/radis.misc.warning.html#radis.misc.warning.AccuracyWarning) or subclass it into a "MissingBroadeningWarning"

Reply:
I don't see the update, did you push?

Reply:
Earlier there was some issue with tests so I didn't push then. I have pushed it now, let me know if it is correct.